demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -667,21 +666,52 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
     }
   }
 
-  async createLogStream(configuration: MeetingSessionConfiguration): Promise<void> {
+  async createLogStream(configuration: MeetingSessionConfiguration, pathname: string): Promise<void> {
     const body = JSON.stringify({
       meetingId: configuration.meetingId,
       attendeeId: configuration.credentials.attendeeId,
     });
     try {
-      const response = await fetch(`${DemoMeetingApp.BASE_URL}create_log_stream`, {
+      const response = await fetch(`${DemoMeetingApp.BASE_URL}${pathname}`, {
         method: 'POST',
         body
       });
       if (response.status === 200) {
-        console.log('Log stream created');
+        this.log('Log stream created');
       }
     } catch (error) {
-      console.error(error.message);
+      this.log(error.message);
+    }
+  }
+  
+  eventDidReceive(name: EventName, attributes: EventAttributes): void {
+    this.log(`Received an event: ${JSON.stringify({ name, attributes })}`);
+    const { meetingHistory, ...otherAttributes } = attributes;
+    switch (name) {
+      case 'meetingStartRequested':
+      case 'meetingStartSucceeded':
+      case 'meetingEnded': {
+        // Exclude the "meetingHistory" attribute for successful events.
+        this.meetingEventPOSTLogger?.info(JSON.stringify({
+          name,
+          attributes: otherAttributes
+        }));
+        break;
+      }
+      case 'audioInputFailed':
+      case 'videoInputFailed':
+      case 'meetingStartFailed':
+      case 'meetingFailed': {
+        // Send the last 5 minutes of events.
+        this.meetingEventPOSTLogger?.info(JSON.stringify({
+          name,
+          attributes: {
+            ...otherAttributes,
+            meetingHistory: meetingHistory.filter(({ timestampMs }) => Date.now() - timestampMs < 5 * 60 * 1000)
```

Comment:
Should we store this `5 * 60 * 1000` in a variable which can be configured by the builders?

Reply:
Addeed it as a static variable: `MAX_MEETING_HISTORY_MS`